tests/ballerina-unit-test/src/test/resources/test-src/expressions/lambda/iterable/iterable-negative.bal

```diff
@@ -67,3 +67,28 @@ function test7(){
     s.filter((person p)=>{});
     _ = s.filter((string s) => (person) {return null;});
 }
+
+function test8() {
```

Comment:
Can we give meaningful names to these functions? If tests are going to document Ballerina we might need meaningful names :)